predeploy.py

```diff
@@ -21,5 +21,23 @@
 else:
     for f in pyfiles:
         shutil.copyfile(os.path.join(app_path, f), f)
+    lines = []
+    ga_line = 0
+    with open(os.path.join(app_path, "app.py"), "r") as f:
+        lines = f.readlines()
+
+    try:
+        ga_line = [i for i in range(len(lines)) if "app.layout" in lines[i]]
+        lines.insert(
+            ga_line[0],
+            "app.index_string = '''\n{}\n'''\n\n".format(os.environ["DASH_GA_CODE"]),
```

Comment:
this is likely not what you want to do, as it will result in no HTML being in `index_string`...